client/client-multi/client-c/src/kaa/kaa_fqn_definitions.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2014-2015 CyberVision, Inc.
```

Comment:
Please, move this header to the gen directory and fix the path used by the SDK generator accordingly.